cumulus_library/cli.py

```diff
@@ -336,14 +354,22 @@ def run_cli(args: Dict):
                 else:
                     for target in args["target"]:
                         builder.export_study(study_dict[target], args["data_path"])
+
+            elif args["action"] == "generate-sql":
+                if "all" in args["target"]:
+                    for target in study_dict.keys():
+                        builder.generate_all_sql(study_dict[target])
+                else:
+                    for target in args["target"]:
+                        builder.generate_study_sql(study_dict[target])
```

Comment:
nit: this is a noiser change, but `builder` as a variable name might be confusing with the class rename.